molsketch/net.sourceforge.molsketch.appdata.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2017 Hendrik Vennekate <mail_AT_domain.tld> -->
+<component type="desktop-application">
+  <id>net.sourceforge.molsketch</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-2.0</project_license>
+  <name>Molsketch</name>
+  <summary>2D Molecular Structures Editor</summary>
+
+  <description>
+    <p>
+    Molsketch is a program for drawing two-dimensional schematic representations of molecules. The goal of the program is to make this quick and easy, without sacrificing the resulting quality. Of course your creations can be used afterwards as vector or bitmap image in any document, presentation or website. Some of the features of Molsketch are:
+    </p>
+    <ul>
+	    <li>Creating regular and consistent bonds easily with tools as a dynamic grid and magnetic atoms</li>
+	    <li>Open, save and import in all formats supported by the OpenBabel™ library</li>
+	    <li>Export to Scalable Vector Graphics (SVG) and a number of common used bitmap formats</li>
+	    <li>Print and export your document to PDF</li>
+	    <li>Intelligent addition/removal of hydrogen atoms</li>
+	    <li>Real-time information about the molecule, like charge and weight</li>
+	    <li>Built-in library for quick access to regularly used molecules and functional groups</li>
+    </ul>
+  </description>
+  
+  <categories>
+	  <category>Science</category>
+	  <category>Chemistry</category>
+  </categories>
+
+  <launchable type="desktop-id">molsketch.desktop</launchable>
+
+  <screenshots>
+    <screenshot type="default">
+      <caption>The Molsketch main window</caption>
+      <image>https://github.com/hvennekate/Molsketch/raw/master/screenshots/Screen1.png</image>
```

Comment:
I have no idea about these appdata files, but are the links supposed to be absolute?

Reply:
My bad, I though I had brought those up at some point.

[Here is the link](https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html) to the official documentation.
And [here](https://fedoraproject.org/wiki/Packaging:AppData) is a handy fedora guide.

The links should point to files hosted someplace and the app providing the "software center" functionality (e.g. GNOME's "Software" or KDE's Apper) downloads and caches them at some point. 

There is also GNOME shell extension for resizing windows to a 16:9 ratio and appropriate size for showcasing programs in software centers. I could use that to take some screenshots if you want me to.

Reply:
Ok, trying this first link, it seems to show a 404... should these images be in _this_ pull request? Or should we rather put them on sourceforge? (don't know, but my intuition tells me that I want github for the actual code...)

Reply:
Hi Hendrik,

>The screenshots declared in the <screenshots> tag do not exist. I thought I'd leave that up to you, so you can choose whatever you feel will look better and you can add/remove captions as you see fit. If you want, you can send me any files you might come up with and I will take some 16:9 screenshots for you.

It's entirely up to you how to deal with them and where to host them. I just added the markup to show you what's possible with the screenshots.

Reply:
Ah, totally forgot about this... I'll have to get back to you on this one.